### PR TITLE
[Perf] MiniMax H3 Qwen3-VL support fused RMSNorm on NPU

### PR DESCRIPTION
**Pull request by wjialish**
PLEASE FILL IN THE PR DESCRIPTION HERE.

## Purpose
Enable fused RMSNorm for the MiniMax H3 Qwen3-VL text encoder on Ascend NPU.

This change replaces the model-local decomposed RMSNorm implementation with
the common diffusion `RMSNorm` implementation, which dispatches to
`torch_npu.npu_rms_norm` on NPU.

The change:

- preserves the Qwen3-VL checkpoint weight key (`weight`);
- preserves BF16 gamma storage used by the checkpoint;
- keeps FP32 accumulation in the native fallback path;
- updates Q/K RMSNorm and pre-/post-attention RMSNorm construction to receive
  the model dtype;
- adds model-level regression coverage for dtype, state-dict compatibility,
  inheritance, and native numerical correctness.

This PR is intentionally scoped to Qwen3-VL RMSNorm only. It does not modify
RoPE, QKV layout, GQA, FlashAttention, DiT, HSDP, or checkpoint naming.

## Test Plan
`export PORT=8078
export MODEL="/data/weights/h3/MiniMax-H3/FL2VA"
export VLLM_WORKER_MULTIPROC_METHOD=spawn
export VLLM_OMNI_VIDEO_SYNC_TIMEOUT=1800
export PYTHONDONTWRITEBYTECODE=1
export HF_HUB_OFFLINE=1
export TRANSFORMERS_OFFLINE=1
export PYTORCH_NPU_ALLOC_CONF=expandable_segments:True

vllm serve "${MODEL}" \
  --omni \
  --host 0.0.0.0 \
  --port "${PORT}" \
  --trust-remote-code \
  --num-gpus 8 \
  --usp 8 \
  --ring 1 \
  --text-encoder-tp-size 8 \
  --enable-layerwise-offload \
  --vae-parallel-mode tile \
  --vae-use-tiling \
  --vae-patch-parallel-size 8 \
  --diffusion-attention-backend FLASH_ATTN \
  --enable-layerwise-offload \
  --use-hsdp`

`export API_URL="http://127.0.0.1:8078/v1/videos/sync"
export FIRST_FRAME="/home/lwj/sample_data/f2lva_sample/first_frame.jpeg"
curl -sS -X POST "${API_URL}" \
  -F 'prompt=A little girl grows up.' \
  -F 'fps=24' \
  -F 'num_inference_steps=50' \
  -F 'flow_shift=12' \
  -F 'seed=2101' \
  -F 'extra_params={"task":"fl2va","duration":8.7,"audio_flow_shift":3.0}' \
  -F "input_reference=@${FIRST_FRAME};type=image/png" \
  -o fl2va.mp4`

image:
image

**vLLM Version:**
`0.26.0`

**vLLM-Omni Commit:**
`f829f4a0`
`[Perf] MiniMax H3 Qwen3-VL support fused RMSNorm on NPU`

### Performance
E2E Latency: 527250ms -> 526587ms. 

ori:
image
opt:
image

### Precision
ori:

https://github.com/user-attachments/assets/a3d8375b-a226-4836-9349-7ad102ba4d24

opt:

https://github.com/user-attachments/assets/abf81646-c59f-4140-84e1-eb246131f74a

**BEFORE SUBMITTING:** read [CONTRIBUTING.md](https://github.com/vllm-project/vllm-omni/blob/main/CONTRIBUTING.md) and run the [precheck-pr skill](https://github.com/vllm-project/vllm-omni/blob/main/.claude/skills/precheck-pr/SKILL.md) with the code agent for a self-check against project conventions.
(anything written below this line will be removed by GitHub Actions)

### CODE REVIEW

**Comment by david6666666** [comment]
please fix pre-commit and DCO

**Review by david6666666 — APPROVED** [review]
LGTM

File: vllm_omni/diffusion/models/minimax_h3/encoder.py [hunk 1]
```diff
@@ -291,18 +293,22 @@ def weight_loader(
 # ---------------------------------------------------------------------------
 
 
-class MiniMaxH3Qwen3VLRMSNorm(nn.Module):
-    def __init__(self, hidden_size: int, eps: float = 1e-6) -> None:
-        super().__init__()
-        self.weight = nn.Parameter(torch.ones(hidden_size))
-        self.variance_epsilon = eps
+class MiniMaxH3Qwen3VLRMSNorm(DiffusionRMSNorm):
+    """Qwen3-VL RMSNorm using the common NPU-fused implementation.
```
Comment (hsliuustc0106) [inline vllm_omni/diffusion/models/minimax_h3/encoder.py]:
```suggestion
    """Qwen3-VL RMSNorm using the common implementation.
```

**Comment by hsliuustc0106** [comment]
CI passed but precommits failed, let me rebase to check

**Comment by hsliuustc0106** [comment]
fix precommits please
